SOURCES/0001-config_change.patch

```diff
@@ -0,0 +1,20 @@
+--- a/src/Makefile	2022-03-31 07:19:49.000000000 +0000
++++ b/src/Makefile	2022-05-06 02:21:27.000000000 +0000
+@@ -41,12 +41,12 @@
+ ENABLE_EEE = y
+ ENABLE_S0_MAGIC_PACKET = n
+ ENABLE_TX_NO_CLOSE = y
+-ENABLE_MULTIPLE_TX_QUEUE = n
+-ENABLE_PTP_SUPPORT = n
+-ENABLE_PTP_MASTER_MODE = n
+-ENABLE_RSS_SUPPORT = n
++ENABLE_MULTIPLE_TX_QUEUE = y
++ENABLE_PTP_SUPPORT = y
++ENABLE_PTP_MASTER_MODE = y
++ENABLE_RSS_SUPPORT = y
+ ENABLE_LIB_SUPPORT = n
+-ENABLE_USE_FIRMWARE_FILE = n
++ENABLE_USE_FIRMWARE_FILE = y
```

Comment:
Why is this change needed? Why do these features need to be enabled? (Please explain like I don't know this hardware... Which is true :)).